$NetBSD: patch-src__tags,v 1.3 2018/11/29 12:04:23 jaapb Exp $

lwt_ppx is now its own package and has been renamed
Use xml-light (patch from upstream)
--- src/_tags.orig	2018-02-08 14:30:46.000000000 +0000
+++ src/_tags
@@ -2,23 +2,23 @@
 true:keep_locs
 
 <lib/type_dir/*.ml{,i}>:eliom_ppx,thread
-<lib/type_dir/*.ml>:package(js_of_ocaml-ppx.deriving,lwt.ppx)
+<lib/type_dir/*.ml>:package(js_of_ocaml-ppx.deriving,lwt_ppx)
 <lib/type_dir/*.ml>:package(js_of_ocaml.ppx)
 
 <lib/server/*.ml{,i}>:eliom_ppx
-<lib/server/*.ml>:package(js_of_ocaml-ppx.deriving,lwt.ppx)
+<lib/server/*.ml>:package(js_of_ocaml-ppx.deriving,lwt_ppx)
 <lib/server/*.ml>:package(js_of_ocaml.ppx)
 
 <lib/server/*.ml{,i}>:thread
 <lib/server/*.ml{,i}>:package(lwt,ocsigenserver,ocsigenserver.ext,tyxml.functor)
-<lib/server/*.ml{,i}>:package(react,lwt_react,js_of_ocaml)
-
+<lib/server/*.ml{,i}>:package(react,lwt_react,js_of_ocaml,lwt_log)
 <lib/server/*.ml{,i}>:package(js_of_ocaml-ppx.deriving)
+<lib/server/*.ml{,i}>:package(xml-light)
 
 <lib/*.eliom{,i}>:eliom_ppx
 
 <lib/*.ml{,i}>:eliom_ppx
-<lib/client/*.ml>:package(js_of_ocaml-ppx.deriving,lwt.ppx,js_of_ocaml-lwt.logger)
+<lib/client/*.ml>:package(js_of_ocaml-ppx.deriving,lwt_ppx,js_of_ocaml-lwt.logger)
 <lib/client/*.ml>:package(js_of_ocaml.ppx)
 
 <lib/client/*.ml{,i}>: eliom_ppx
@@ -29,7 +29,7 @@ true:keep_locs
 
 <lib/client/*.ml{,i}>:package(js_of_ocaml-ppx.deriving)
 
-<lib/server/monitor/*.ml>:package(lwt.ppx)
+<lib/server/monitor/*.ml>:package(lwt_ppx)
 <lib/server/monitor/*.ml{,i}>:thread
 <lib/server/monitor/*.ml{,i}>:package(lwt,ocsigenserver,ocsigenserver.ext,tyxml)
 <lib/server/monitor/*.ml{,i}>:I(src/lib/server)
